jaxb/src/main/java/feign/jaxb/JAXBContextFactory.java

```diff
@@ -126,5 +133,20 @@ public Builder withMarshallerFragment(Boolean value) {
     public JAXBContextFactory build() {
       return new JAXBContextFactory(properties);
     }
+
+    /**
+     * Creates a new {@link feign.jaxb.JAXBContextFactory} instance.
+     * Pre-loads context cache with given classes
+     * @param classes
+     * @return ContextFactory with a prep-populated JAXBContext cache
+     * @throws JAXBException
+     */
+    public JAXBContextFactory buildWithClasses(List<Class<?>> classes) throws JAXBException {
```

Comment:
I think we should name this method `build` and document it's usage.

```java
build(); // builds a lazy loading cached context
build(List ...) // builds a context with an initial set of classes cached.
```

Make sense?